### PR TITLE
Fix for nxos smu issue

### LINKED ISSUE #115: nxos_smu.py invalid character :(58) found in input when activating and committing smu
_From @oh-c (Copied from original issue: https://github.com/ansible/ansible/issues/67570):_

##### SUMMARY
In relation to CSCvs27997 i've tried using nxos_smu.py to install, activate and commit the SMU.

Installation runs fine, but activate and commit fails but the module reports that everything is OK.

```
Hardware
  cisco Nexus7700 C7710 (10 Slot) Chassis ("Supervisor Module-3")
  System version: 8.4(1)
```

##### ISSUE TYPE
- Bug Report

##### COMPONENT NAME
nxos_smu.py

##### ANSIBLE VERSION
```
ansible 2.9.5
```

##### STEPS TO REPRODUCE

```yaml
- hosts: all
  gather_facts: true
  connection: network_cli

  tasks:

  - name: Install SMU
    nxos_smu:
      pkg: n7700-s3-dk9.8.4.1.CSCvs27997.bin
```

##### EXPECTED RESULTS
package installed, activated and committed

##### ACTUAL RESULTS

package was installed but not commited.

```
changed: [sw-01] => {
    "changed": true,
    "commands": [
        "install add bootflash:n7700-s3-dk9.8.4.1.CSCvs27997.bin",
        "install activate bootflash:n7700-s3-dk9.8.4.1.CSCvs27997.bin force",
        "install commit bootflash:n7700-s3-dk9.8.4.1.CSCvs27997.bin"
    ],
    "invocation": {
        "module_args": {
            "auth_pass": null,
            "authorize": null,
            "file_system": "bootflash:",
            "host": null,
            "password": null,
            "pkg": "n7700-s3-dk9.8.4.1.CSCvs27997.bin",
            "port": null,
            "provider": null,
            "ssh_keyfile": null,
            "timeout": null,
            "transport": null,
            "use_ssl": null,
            "username": null,
            "validate_certs": null
        }
    }
}
```

If i reproduce the commands on the switch itself i get these errors

```
sw-01# install activate bootflash:n7700-s3-dk9.8.4.1.CSCvs27997.bin force
invalid character :(58) found in input
Command contains invalid character
sw-01# install commit bootflash:n7700-s3-dk9.8.4.1.CSCvs27997.bin
invalid character :(58) found in input
```

Seems it's bothered by the bootflash path. If i remove bootflash:

```
sw-01# install activate n7700-s3-dk9.8.4.1.CSCvs27997.bin
Install operation 2 completed successfully at Wed Feb 19 12:45:25 2020
sw-01# install commit n7700-s3-dk9.8.4.1.CSCvs27997.bin
Install operation 3 completed successfully at Wed Feb 19 12:45:40 2020
```

### PR DESCRIPTION
**Pull request by praveenramoorthy**
##### SUMMARY

Fixes #115 
The smu module passes smu path along with smu name for activate and commit steps and is not required.

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME

modules/nxos_smu.py

### CODE REVIEW

**Comment by NilashishC** [comment]
Hi Praveen. Thank you for the PR. The validate-modules sanity test is [failing](https://1dcbb2fd450b06dd439f-378a68fa3e56314ec6475a02a3956352.ssl.cf2.rackcdn.com/160/5d528811c7f22a03c7dace44bd9e8b2c9aba1b06/check/ansible-test-sanity-nxos/608bfa2/job-output.html#l37976). This is because Ansible 2.9 doesn't support the new deprecation process and our sanity jobs are run with 2.9 as well. Hence, you would need to add ignores for this check [here](https://github.com/ansible-collections/cisco.nxos/blob/main/tests/sanity/ignore-2.9.txt).

File: changelogs/fragments/fix_nxos_smu.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+---
```
Comment (NilashishC) [inline changelogs/fragments/fix_nxos_smu.yaml:1]: Could you also please add the following?
```yaml
deprecated_features:
  - Deprecated `nxos_smu` in favor of `nxos_rpm` module.
```

Comment (praveenramoorthy) [reply changelogs/fragments/fix_nxos_smu.yaml:1]: Done

**Comment by NilashishC** [comment]
recheck

**Comment by GomathiselviS** [comment]
recheck

**Comment by GomathiselviS** [comment]
recheck

**Comment by GomathiselviS** [comment]
recheck

**Comment by NilashishC** [comment]
recheck

**Pull request merged**